runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/VariablesActions.ts

```diff
@@ -75,7 +75,7 @@ const variablesActions: Readonly<Record<string, IBpmnAction>> = {
             formData: {
                 input: {
                     variable: undefined,
-                    value: undefined,
+                    value: [''],
```

Comment:
what happened here?

Reply:
Because we apply action `on creation` there was app breaking error, this is temporary workaround and im working on better solution